### PR TITLE
Add landing message #1

### PR DESCRIPTION
**Pull request by rbormann001**
I added a title

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Modern Cloud</title>
+    <title>Modern Cloud Project</title>
 </head>
 <body>
```
Comment (Rouvx) [inline index.html]: add an h1 object on line 10

**Review by Rouvx — COMMENTED** [review]
add an h1 object

File: index.html (hunk 1 above)
Comment (rbormann001) [reply index.html]: I will add the h1

**Review by Rouvx — CHANGES_REQUESTED** [review]
submitting change request

**Review by Rouvx — APPROVED** [review]
Good work!

**Pull request merged**